Cargo.lock

```diff
@@ -13189,6 +13582,26 @@ dependencies = [
  "wasm-opt",
 ]
 
+[[package]]
+name = "substrate-wasm-builder"
+version = "22.0.0"
```

Comment:
This is the new one we get through the update, we have to find out where we still have the old one.